contributors/devel/kubemark-guide.md

```diff
@@ -13,27 +13,35 @@ and how to use it.
 
 ## Architecture
 
-On a very high level Kubemark cluster consists of two parts: real master
-components and a set of “Hollow” Nodes. The prefix “Hollow” means an
-implementation/instantiation of a component with all “moving” parts mocked out.
+On a very high level, Kubemark cluster consists of two parts: a real master
+and a set of “Hollow” Nodes. From now on, the prefix “Hollow” to any component
+means an implementation/instantiation of the actual component with all “moving”
+parts mocked out. Essentially, a dummy component which just sends requests and
+receives responses but does no real work i.e. the internal logic is hollow.
```

Comment:
That's not entirely true. In particular HollowKubelet is not just sending requests and receiving responses. It does a lot of real work, but managing pod lifecycles, managing secrets, etc. The only difference is that we are faking Docker, thus containers are reported to be running whereas they are not really running. 
So it __does__ real work (or at least some parts of it).